src/stk/io/mod.rs

```diff
@@ -0,0 +1,71 @@
+use emdrv::gpioint;
+use {cmu, gpio};
+
+pub struct Button {
+    port: gpio::Port,
+    pin: u8,
+}
+
+impl Button {
+
+    fn new(port: gpio::Port, pin: u8) -> Button {
+        Button {
+            port: port,
+            pin: pin,
+        }
+    }
+
+    pub fn init(&self) {
+        cmu::clock_enable(cmu::Clock::GPIO, true);
+        gpio::pin_mode_set(self.port, self.pin as u32, gpio::Mode::Input, 0);
+        gpio::int_config(self.port, self.pin as u32, false, true, true);
+
+        gpioint::init();
+    }
+
+    pub fn pb0() -> Button {
+        Button::new(gpio::Port::B, 9)
+    }
+
+    pub fn pb1() -> Button {
```

Comment:
Maybe these functions can both initialize _and_ return a Button struct?

This will reduce the code in the example to 
```
    let btn0 = Button::pb0();
    let btn1 = Button::pb1();

    Led::led0().init();
    Led::led1().init();

    btn0.click(led0);
    btn1.click(led1);
```
If so, it would probably make more sense to rename the `led0`and `led1` functions to `init_led0` and `init_led1` or something like that?